### ISSUE TITLE
Announcement played

### ISSUE THREAD

**Issue by f59465-Clem** [talkkonnect/talkkonnect#83]
Good morning,
Using talkconnect as an RF/VoIP gateway.

I want to broadcast an announcement (.wav file, gateway relay active) about 10s after Talkkonnect connects to the server, or with http API

I want this audio announcement to be played on the RF side only, not Voip, just to inform users that the radio relay is active.
Or use the beacon function only on the RF side.

Is it possible?

Thanks
Clem

**Comment by talkkonnect** [comment]
Yes this should be possible with http api. But there is nothing like after connecting 10 seconds then play. It is adhoc by httpapi only currently.

**Comment by f59465-Clem** [comment]
ok Suvir,
I will study http api, and make a bash script with the command.
I would return.

Thank you for the new Features, I have not yet understood how to update without destroying my .img
73'

**Comment by Propeller-v8** [comment]
Clem,

update routine:
To update to the lastest version release of talkkonnect in the image you can:
cd to the root directory

> rm update-talkkonnect.sh*
> wget https://raw.githubusercontent.com/talkkonnect/talkkonnect/main/scripts/update-talkkonnect.sh
> chmod +x ./update-talkkonnect.sh
> /root/update-talkkonnect.sh

Easy
greets Andre

**Comment by f59465-Clem** [comment]
404 not found...

**Comment by talkkonnect** [comment]
Clem,

have a look here https://github.com/talkkonnect/talkkonnect/tree/main/scripts
choose the update script that matches your use case. See the original image if 32 or 64 bit os.
Click on the image then click on raw in your browser, once that is done copy the raw url of the desired script.
Then use Andre Steps as above changing the URL to the appropriate one you just selected.

I have updated the update scripts to update golang also as the new versions need the latest version of golang.
Using the new correct scripts the build will not break the upgrade and talkkonnect should work as expected after
upgrading.

**Comment by f59465-Clem** [comment]
Suvir,

In the meantime I have adapted the command according to my image (PiZ2x, 64 bits, USB sound card):

rm update-talkkonnect.sh*
wget https://raw.githubusercontent.com/talkkonnect/talkkonnect/main/scripts/update-talkkonnect-64bit.sh
chmod +x ./update-talkkonnect-64bit.sh
/root/update-talkkonnect-64bit.sh

the talkkonnect.xml file was empty so I copied the latest version.

**Comment by talkkonnect** [comment]
So you successfully updated?

**Comment by f59465-Clem** [comment]
Yes successfully update .

**Comment by f59465-Clem** [comment]
To return to the original subject of the Announcement played station on the RF side with the voiceactivity GPIO set to high, which sends the audio from a .wav file to the radio.

The goal is to send a looping audio file every x minutes on the Radio RF side for an announcement intended to inform users on radio waves only.

In the API I can't find a command to activate the voiceactivity GPIO, nor a command to broadcast an audio directly, apart from playback, but it needs two actions.
 
I thought of modifying the beacon function, keeping the same setting parameters and authorizing sending only on the RF side (GPIO voiceactivity).
Follow, did you have a trick to do this or is it more subject to a request for a new function ?

Thanks

**Comment by talkkonnect** [comment]
To play a looping file every x minutes use beacon function. No need api.

**Comment by f59465-Clem** [comment]
Hi Suvir,

So I did a test with "Beacon", unsurprisingly the beacon function works, on the other hand subtlety it sends the audio only on the mumble side (gpio txptt) not on the RF side (gpio voiceactivity).

Is there a possibility to modify this?
Or add an option in a future release to choose gpio txptt or voiceactivity or both?

Concerning "multimedia section" I do not understand how it works I did not find any help on this subject, how to make it work ?

**Comment by f59465-Clem** [comment]
Hi Suvir,

I'll give you a feedback on the use of the beacon function locally.

 Overall it works, however there is a weird operation, if the beacon is triggered while a person is talking from Mumble, the beacon will be played at the same time ok, however at the end of the beacon it cuts the audio on the mumble side, hang very long until a new operator speaks. 

I'm not sure it's clear.

I think we should put a condition on the audio tag, if audio since mumble no tag played until next time.

Cordially
Clem
